### ISSUE TITLE
Adding library to dotnet core project

### ISSUE THREAD

**Issue by ronaksharma8** [libgit2/libgit2sharp#1435]
I am getting the following error while adding library via package manage console in .Net Core project. Can someone plz shred some light on whether this "LibGit2Sharp" library is compatible with .Net Core project.

**Error**

_Sequence contains more than one matching element_

**Comment by bording** [comment]
@ronaksharma8 LibGit2Sharp doesn't work with .NET Core yet, but we're getting close to having a version that can: #1138

**Comment by ethomson** [comment]
What @bording said is correct.  If you're using a nuget package, check to include prerelease packages and install the LibGit2Sharp.Portable.  It's been working for me so far!

**Comment by Platzer** [comment]
I added this reference, to my `.csproj`:
```
  ...
  <ItemGroup>
    <PackageReference Include="LibGit2Sharp.Portable" Version="0.24.10" />
  </ItemGroup>
  ...
```
but getting still this error: 
```
Unhandled Exception: System.TypeLoadException: Could not load type 'LibGit2Sharp.Repository' from assembly 'LibGit2Sharp, Version=1.0.0.0, Culture=neutral, Pub
licKeyToken=null'.
   at LibGit2Sharp.Program.Main(String[] args)
```
```
PS C:\temp\dotnet\LibGit2Sharp> dotnet --info
.NET Command Line Tools (1.0.1)

Product Information:
 Version:            1.0.1
 Commit SHA-1 hash:  005db40cd1

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.1
```
Did i referenced the wrong package?

**Comment by ethomson** [comment]
Fixed via #1138 - @Platzer if you're having problems with that then please open a new issue!  😁
